hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -1004,34 +1002,23 @@ onOpenChainIncrementTx openState newChainState newVersion depositTxId =
 --
 -- __Transition__: 'OpenState' → 'OpenState'
 onOpenChainDecrementTx ::
-  IsTx tx =>
   OpenState tx ->
   ChainStateType tx ->
   -- | New open state version
   SnapshotVersion ->
   -- | Outputs removed by the decrement
-  [TxOutType tx] ->
+  UTxOType tx ->
   Outcome tx
-onOpenChainDecrementTx openState newChainState newVersion distributedTxOuts =
-  -- Spec: if outputs(txω) = 𝑈ω
```

Comment:
question: so this has to be removed from spec?

Reply:
https://github.com/cardano-scaling/hydra/pull/1928#discussion_r2034648428